themes/default/content/docs/intro/console/ci-cd-integration-assistant.md

```diff
@@ -57,8 +60,10 @@ Services such as Atlassian Bitbucket, GitHub, GitLab offer both a version contro
 
 This might be a convenient option if your team wishes to keep everything related to your Pulumi project on a single service.
 
-> If your identity isn't tied to a VCS service, like SAML or Email, you will still be able to get a customized CI/CD
-> workflow for your stack.
+{{% notes "info" %}}
+If your identity isn't tied to a VCS service, like SAML or Email, you will still be able to get a customized CI/CD
```

Comment:
This wording is awkward, as it sounds like "SAML or Email" are examples of VCS services.

```suggestion
If your identity isn't tied to a VCS service (for example, if you're using SAML or email), you will still be able to get a customized CI/CD
```